### ISSUE TITLE
Inconsistencies in email addresses in make enquiry form

### ISSUE THREAD

**Issue by johnedwarddalton** [propertyhive/WP-Property-Hive#209]
In the function ```make_property_enquiry``` in ```class_ph-ajax.php``` the following line (1984) forms the "From" email header:

```$headers[] = 'From: ' . html_entity_decode(ph_clean( $_POST['name'] )) . ' <' . sanitize_email( $from_email_address ) . '>';```

However, the variable ```$from_email_address``` is defined a few lines earlier as the ```propertyhive_email_from_address``` which is defined in the PropertyHive settings (or else the ```admin email``` if the former is not defined.  

This means that the enquirer's name is being associated with the general office email address, which is clearly not correct.  

In addition, the "Reply to" header is being set as the enquirer's email address which I'm not sure is the correct approach. In effect, the email that is created should be a message from the website to the office manager, so the "From" and "Reply to" headers should reflect that.

I am flagging this because I have recently had some issues on my shared (GoDaddy) server where enquiries were not being sent and I believe it is related to GoDaddy's anti-spam algorithms.

**Comment by propertyhive** [comment]
@johnedwarddalton Thanks for getting in touch. I believe setting the `Reply-To` as the enquirers email address is correct as, in doing this, should the agent hit 'Reply' their end, the default email address is the enquirers. Otherwise they'd have to manually change this each time.

In terms of the `From` address, we used to use the enquirers email address but changed this years ago in version 1.2.8:

https://github.com/propertyhive/WP-Property-Hive/commit/837956397635a9c7330c1a79330cea81060396b3

The reason being that enquiries where getting rejected a lot because we were essentially masking the `from` domain which caused lots of issues and SMTP servers and spam filters didn't like this.

Do you use any kind of SMTP plugin on your site out of interest?

Steve

**Comment by johnedwarddalton** [comment]
Understood.  However, the "From" address still has an issue as it lists the enquirer's name next to the admin (```from_email_address```) email address.  (So in the email I receive I would see "From:  Joe Public <ph-admin@mycompany.com">).  I would suggest that the ```html_entity_decode(ph_clean( $_POST['name'] ))``` be removed.

We don't use any SMTP add-ins and the bug I was referring to has since disappeared, suggesting that it was a temporary issue with GoDaddy's mail servers.
